core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/aiservice/AiServiceMethodCreateInfo.java

```diff
@@ -193,6 +193,13 @@ public OutputTokenAccumulator getOutputTokenAccumulator() {
         return accumulator;
     }
 
+    public String getUserMessageTemplate() {
+        Optional<String> userMessageTemplateOpt = this.getUserMessageInfo().template()
+                .flatMap(AiServiceMethodCreateInfo.TemplateInfo::text);
+
+        return userMessageTemplateOpt.orElse(null);
```

Comment:
@geoand would it be better to use `null` or a blank or a sentinel value?

Reply:
I believe `null`

Reply:
I going to change it to be empty instead of null.